src/libraries/System.Reflection/tests/TypeTests.Constraints.cs

```diff
@@ -0,0 +1,165 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is copy of the test from System.Reflection.MetadataLoadContext with modifications for runtime reflection (all `Project()` calls deleted, irrelevant test deleted).